tool/dartpad_picker/web/dartpad_picker_main.dart

```diff
@@ -52,13 +52,13 @@ bool isMobile() {
 
 // Snippets
 
-var helloWorld = r'''
-main() {
+const helloWorld = r'''
+void main() {
   print("Hello, World!");
 }
-'''
-    .trim();
-var functions = r'''
+''';
```

Comment:
Is trimming not necessary?

Reply:
It changed the extra behavior of the line at the end, but in my latest commit I've moved the multi-line literal right after the previous bracket to match the earlier behavior so the strings are equivalent while still allowing them to be const.

Reply:
cool, LGTM